sources/packages/api/src/route-controllers/note/models/note.dto.ts

```diff
@@ -1,15 +1,19 @@
+import { IsNotEmpty } from "class-validator";
+
 /**
  * Base DTO for note.
  */
-export interface NoteBaseDTO {
+export class NoteBaseAPIInDTO {
+  @IsNotEmpty()
```

Comment:
Is is named Base dto because it is extended by the out dto? If so we can rename this to NoteAPIInDTO.